src/ui/public/vis/editors/default/default.js

```diff
@@ -2,11 +2,12 @@ import './sidebar';
 import './vis_options';
 import $ from 'jquery';
 
-
 import _ from 'lodash';
 import angular from 'angular';
 import defaultEditorTemplate from './default.html';
 
+import { VisEditorTypesRegistryProvider } from 'ui/registry/vis_editor_types';
+
```

Comment:
Importing the module won't cause the code to execute twice. As long as the register call is only in one place it will be fine.